doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,126 @@
+[id='spring-boot-jms-audit-proc_{context}']
+= Replicating audit data in a JMS message broker
+
+You can replicate {KIE_SERVER} audit data to a Java Message Service (JMS) message broker, for example ActivMQ or Artemis, and then dump the data in an external database schema so that you can improve the performance of your Spring Boot application by deleting the audit data from your application schema.
+
+If you configure your application to replicate data in a message broker, when an event occurs in {KIE_SERVER} the record of that event is stored in the {KIE_SERVER} database schema and it is sent to the message broker. You can then configure an external service to consume the message broker data into an exact replica of the application's database schema. The data is appended in the message broker and the external database every time an event is produce by {KIE_SERVER}.
```

Comment:
I may be wrong her, but wouldn't another comma make the 1st sentence flow better?

If you configure your application to replicate data in a message broker, when an event occurs in {KIE_SERVER}, the record of that event is stored in the {KIE_SERVER} database schema and it is sent to the message broker.

Reply:
Yes. I see the difference. Thank you Emily.